### ISSUE TITLE
"Applied Accessibility - Make Screen Reader Navigation Easier with the header Landmark" cannot be completed because it always thinks there are <div>s

### ISSUE THREAD

**Issue by SuperGeniusZeb** [freeCodeCamp/learn#128]
https://learn.freecodecamp.org/responsive-web-design/applied-accessibility/make-screen-reader-navigation-easier-with-the-header-landmark

This lesson cannot be completed because one of the tests ("Your code should not have any `div` tags.") always fails, no matter what the content of the editor is.

Tested in Firefox Nightly.

Related:
- #127
- #129
- #130
- #134
- #136

**Comment by Bouncey** [comment]
I cannot reproduce, can you share your code please?

**Comment by SuperGeniusZeb** [comment]
```
<body>

  <header>
    <h1>Training with Camper Cat</h1>
  </header>


  <main>
    <section id="stealth">
      <h2>Stealth &amp; Agility Training</h2>
      <article><h3>Climb foliage quickly using a minimum spanning tree approach</h3></article>
      <article><h3>No training is NP-complete without parkour</h3></article>
    </section>
    <section id="combat">
      <h2>Combat Training</h2>
      <article><h3>Dispatch multiple enemies with multithreaded tactics</h3></article>
      <article><h3>Goodbye world: 5 proven ways to knock out an opponent</h3></article>
    </section>
    <section id="weapons">
      <h2>Weapons Training</h2>
      <article><h3>Swords: the best tool to literally divide and conquer</h3></article>
      <article><h3>Breadth-first or depth-first in multi-weapon training?</h3></article>
    </section>
  </main>
</body>
```
The `div` test also fails if there is nothing in the editor.

I just tested in Chromium, and the issue is not happening there. Can you reproduce the issue in Firefox?

Also, I have noticed that the website switched back and forth between 2 designs yesterday. Does the website look like this to you right now?
![image](https://user-images.githubusercontent.com/19592990/40842621-153344d2-6574-11e8-95a4-9871f4040928.png)

(The other design has a second row under the header with the <kbd><samp>Run the Tests</samp></kbd> button in it, and the tests show up under the preview of the project instead of the description.)

**Comment by raisedadead** [comment]
https://github.com/freeCodeCamp/learn/issues/136#issuecomment-398936869
